### DISCUSSION TITLE
Projects using model based testing ?

### DISCUSSION THREAD

**Discussion by gugakatsi** [dubzzz/fast-check#4318] [Q&A]
Any projects using model-based testing? 

I think it would be good for documentation as well.

**Comment by dubzzz** [comment]
On my side I have this one https://github.com/dubzzz/connect-four-react and a few examples in the documentation. I used it in rare cases on fast-check itself and mostly used pure property based testing to test it (including variants with race conditions).

I forgot about [this ticket](https://github.com/dubzzz/fast-check/issues/3844) but when I migrated to my new website, I initially planned to better document model based testing with blog posts, dedicated examples... So far it's limited to a few pages in the documentation and some examples (like [this one](https://github.com/dubzzz/fast-check/tree/main/examples/004-stateMachine/musicPlayer) or [this one](https://github.com/dubzzz/fast-check/tree/main/examples/005-race/todolist)).

**EDITED:** Anyway, having/knowing a list of projects using it will be cool! Even for raw property based tests. I have [this page](https://github.com/dubzzz/fast-check/discussions/1540) where people can show off their projects and the usage they do of fast-check.

**Reply by gugakatsi** [reply]
@dubzzz not related to examples, but "Recursive Structure" could have better documentation, if I could get quickly how it works internally, I have an infinite generation problem for example now and have to dig into code etc to solve the problem, Will re-read documentation for it tomorrow and will make issue if needed.

**Reply by dubzzz** [reply]
Note: Thanks, please or anything like that is always welcomed 🤗
